python/triton/ops/blocksparse/softmax.py

```diff
@@ -11,8 +11,8 @@ def num_warps(n):
     return 16
 
 
-@triton.heuristics({'num_warps': lambda *args, **meta: num_warps(args[7] * meta['BLOCK'])})
-@triton.heuristics({'TN': lambda *args, **meta: triton.next_power_of_2(args[7] * meta['BLOCK'])})
+@triton.heuristics({'num_warps': lambda nargs, **meta: num_warps(nargs['sizemax'] * meta['BLOCK'])})
+@triton.heuristics({'TN': lambda nargs, **meta: triton.next_power_of_2(nargs['sizemax'] * meta['BLOCK'])})
```

Comment:
Probably `triton.heuristics` doesn't need `meta` anymore with Triton 2.0, now that all arguments are named

Reply:
Yep! I think so